### ISSUE TITLE
Feature Request: Export / Save profile(s)

### ISSUE THREAD

**Issue by Spiffo** [mendhak/gpslogger#746]
Allow to export settings saved under a specific profile, so that you don't have to go through the entire setup again after getting a new phone. Options would be to save it to a separate file (on dropbox, drive, sftps, etc.) like you allow to send log files.

**Comment by StephenAtty** [comment]
I'd love to see this too. I have a website that offers tracking and having the ability to create a config file for a user so they can download it to their device and then load it would be a great feature to have.

**Comment by StephenAtty** [comment]
It looks like this could be related : 

#567

**Comment by mendhak** [comment]
Work in progress:  https://github.com/mendhak/gpslogger/pull/760

If you can help with testing that would be great

**Comment by StephenAtty** [comment]
I'll put my spare phone into developer mode and get the file side loaded. I'll grab the config file that I've saved and put it onto my webserver and make the required adjustments to get it to download.

It might be a few days before I can do it because I'm on call at work this coming week.

**Comment by StephenAtty** [comment]
OK I changed the settings on my phone and side loaded it.

Reconfigured my website so it would present a button which calls gpslogger://properties/ with my configuration URL on the end (I use a php file to build custom config files).

Went to my website on the phone and clicked the link and it downloaded the configuration. 

Went into the app and it seemed to have loaded the configuration.

Started logging and it pushed a location update through to my website:

![Screenshot_2019-07-07_19-42-04](https://user-images.githubusercontent.com/251449/60772503-5aa68080-a0ef-11e9-9c73-dbb91e212769.png)

I also removed the profile and then loaded the configuration using the load from URL option and that seemed to work too.

I'll play with it a little more over the next few days to make sure its working.
